clusterman/autoscaler/pool_manager.py

```diff
@@ -206,11 +206,12 @@ def get_node_metadatas(self, state_filter: Optional[Collection[str]] = None) ->
             for instance_metadata in group.get_instance_metadatas(state_filter)
         ]
 
+    # currently dead code, so don't count towards coverage metrics
     def _filter_scale_up_options_for_pod(
         self,
         pod: KubernetesPod,
         scale_up_options: Mapping[str, List[ClusterNodeMetadata]],
-    ) -> Mapping[str, List[ClusterNodeMetadata]]:
+    ) -> Mapping[str, List[ClusterNodeMetadata]]:  # pragma: no cover
         filtered_options: Mapping[str, List[ClusterNodeMetadata]] = defaultdict(list)
         for group_id, options in scale_up_options.items():
             for option in options:
```

Comment:
why not just delete it?

Reply:
In the _super ideal_ world this code would become un-dead at some point in the future.